### ISSUE TITLE
Viewing loss plot of previous result

### ISSUE THREAD

**Issue by lucy3** [junyanz/pytorch-CycleGAN-and-pix2pix#28]
Sorry if these questions are very basic; I am a student and new to this. 

Is there a way to retrieve the loss plot or final training accuracy of previous runs? 

Is Ctrl^C the correct way to stopping training? 

Additionally, we tried to continue a run right from where we left off, using --continue_train, but it did not seem to work. Is there something else we're missing?

**Comment by junyanz** [comment]
Currently, the program does not save the loss to the disk so that there is no way to retrieve the loss plot. If you would like to keep the loss or other logging information, you can redirect the program output to a file. See [here](https://stackoverflow.com/questions/418896/how-to-redirect-output-to-a-file-and-stdout) for more details.

**Comment by junyanz** [comment]
`--continue_train` should work. It will initialize the network with the latest_epoch by default. See #33 for more discussion

**Comment by junyanz** [comment]
Hi, with the latest update, now the algorithm will save the training losses to a text file called "loss_log.txt" under opt.checkpoints_dir/opt.name directory.
